website_portal_comment_attachment/static/src/js/portal_chatter.js

```diff
@@ -0,0 +1,67 @@
+odoo.define('website_portal_comment_attachment.chatter', function (require) {
+    'use strict';
+    var core = require('web.core');
+    var ajax = require('web.ajax');
+    var PortalChatter = require('portal.chatter').PortalChatter;
+
+    PortalChatter.include({
+        _loadTemplates: function(){
+            var res = this._super.apply(this, arguments);
+            ajax.loadXML('/website_portal_comment_attachment/static/src/xml/portal_chatter.xml', core.qweb);
+            return res;
```

Comment:
```res``` is a deferred object, use it:
```suggestion
            return this._super.apply(this, arguments).then(function () {
                ajax.loadXML('/website_portal_comment_attachment/static/src/xml/portal_chatter.xml', core.qweb);
            }
```

Reply:
Can you add the final parenthesis ? in order i add you suggestion to the batch commit. ^^

Reply:
Tested : defering `ajax.loadXML` (after _loadTemplate), avoid the view to be used...